front/pages/w/[wId]/join.tsx

```diff
@@ -66,23 +128,49 @@ export default function Join({
             </p>
           </div>
           <div className="h-10"></div>
-          <div>
-            <p className="font-regular mb-16 text-slate-400">
-              Glad to see you!
-              <br />
-              Please log in or sign up with your company email to access this
-              page.
-            </p>
-            <GoogleSignInButton
-              onClick={() =>
-                signIn("google", {
-                  callbackUrl: `/api/login?wId=${wId}&cId=${cId}&join=true`,
-                })
-              }
-            >
-              <img src="/static/google_white_32x32.png" className="h-4 w-4" />
-              <span className="ml-2 mr-1">Sign in with Google</span>
-            </GoogleSignInButton>
+          <div className="font-regular text-lg text-slate-200">
+            <p>Glad to see you!</p>
+
+            {onboardingType === "domain_conversation_link" ? (
+              <p>
+                Please log in or sign up with your company email to access this
+                conversation.
+              </p>
+            ) : (
+              <p>
+                You've been invited to join the Dust workspace of{" "}
+                {workspaceName}.
```

Comment:
We're now displaying the name of the workspace on the join page for non-logged-in users. Privacy-wise, are we okay with that?